hack/update-catalogs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export OPERATOR_BUNDLE_IMAGE="registry.redhat.io/cert-manager/cert-manager-operator-bundle@sha256:f3324ea2051d4ffb00136de53f5355bcc0449bc79349833ddaae528cb37cd3b0"
+export BUNDLE_FILE_NAME="bundle-v1.16.0.yaml"
+
+make update-catalog CATALOG_DIR=catalogs/v4.14/catalog REPLICATE_BUNDLE_FILE_IN_CATALOGS=no USE_MIGRATE_LEVEL_FLAG=no
```

Comment:
Nice! Once support for 4.14, 4.15, 4.16 ends, we can set `REPLICATE_BUNDLE_FILE_IN_CATALOGS` to yes to copy the same to all versions.

Reply:
Yeah, whenever we follow-up with that we can also turn this `hack/update-catalogs.sh` into a proper make target 🙂